app/views/comments/_comment_date.erb

```diff
@@ -6,8 +6,11 @@
   </time>
 
   <% if decorated_comment.edited_at.present? %>
-    <%= t("views.comments.edited.text_html", 
-          on: tag.span(t("views.comments.edited.on_html", date: tag.time(decorated_comment.edited_at.strftime(t("time.formats.short")), datetime: decorated_comment.edited_timestamp, class: %w[hidden m:inline-block date-no-year])))
-    ) %>
+    <span class="hidden m:inline-block"><%=
```

Comment:
@yheuhtozr I am surprised that this has the `hidden` class now, can you explain a bit of your thinking here?

Reply:
I didn't add it. I just transferred it from the `<time>` element to the container. In this app's CSS `hidden m:inline-block` seems to mean "hidden in mobile view and displayed as inline-block in desktop" and `m:hidden` at line 14 does the opposite.